src/components/InterviewRejectReasonDropdown/InterviewRejectReasonDropdown.i18n/en.json

```diff
@@ -1,3 +1,3 @@
 {
-    "Standard Comment": ""
```

Comment:
Why was standard comment removed?

Reply:
Because we not only have a standard comment, but also have our own comment.